### PR TITLE
Rector fixes from rule set LevelSetList::UP_TO_PHP_81

### PR DESCRIPTION
**Pull request by Harfusha**
Made new PR for 5.next branch. Original PR: https://github.com/cakephp/cakephp/pull/17758

### CODE REVIEW

File: tests/test_app/templates/Pages/home.php [hunk 1]
```diff
@@ -81,7 +81,7 @@
 <?php endif; ?>
 
 <p>
-<?php if (Plugin::loaded('DebugKit')): ?>
+<?php if (Plugin::loaded()): ?>
```
Comment (ADmad) [inline tests/test_app/templates/Pages/home.php]:
```suggestion
<?php if (Plugin::isLoaded('DebugKit')): ?>
```

File: tests/test_app/TestApp/Model/Entity/NonExtending.php [hunk 2]
```diff
@@ -35,7 +35,7 @@ public function __construct(array $properties = [], array $options = [])
         }
 
         if ($options['markNew'] !== null) {
-            $this->isNew($options['markNew']);
+            $this->isNew();
```
Comment (ADmad) [inline tests/test_app/TestApp/Model/Entity/NonExtending.php]:
```suggestion
            $this->setNew($options['markNew']);
```

**Comment by Harfusha** [comment]
Should i add those in this PR when the functionality is still the same (It changes the behaviour correctly, but still changes it)

File: tests/test_app/TestApp/Model/Entity/NonExtending.php (hunk 2 above)
Comment (Harfusha) [reply tests/test_app/TestApp/Model/Entity/NonExtending.php]: Should i add this in this PR when the functionality is still the same? (It changes the behaviour correctly, but still changes it - test was not working as intended)

File: tests/test_app/templates/Pages/home.php (hunk 1 above)
Comment (Harfusha) [reply tests/test_app/templates/Pages/home.php]: Should i add this in this PR when the functionality is still the same? (It changes the behaviour correctly, but still changes it - test was not working as intended)

Comment (ADmad) [reply tests/test_app/templates/Pages/home.php]: Yes, include both the method name fixes in this PR itself.

**Review by ADmad — APPROVED** [review]

**Comment by Harfusha** [comment]
How does this look? Should i split it into multiple PRs or can i keep it?

**Comment by ADmad** [comment]
Just resolve the merge conflicts.

**Comment by Harfusha** [comment]
This PR is missing in the issue timeline, will add it: https://github.com/cakephp/cakephp/issues/17756

**Review by othercorey — APPROVED** [review]
Looks good.

**Pull request merged**